README.md

```diff
@@ -1,122 +1,163 @@
-reqDataFrame.RegisterTempTable("requests");
-metricDataFrame.RegisterTempTable("metrics");
-// C0 - guid in requests DataFrame, C3 - guid in metrics DataFrame  
-var joinDataFrame = GetSqlContext().Sql(  
-    "SELECT joinedtable.datacenter" +
-         ", MAX(joinedtable.latency) maxlatency" +
-         ", AVG(joinedtable.latency) avglatency " + 
-    "FROM (" +
-       "SELECT a.C1 as datacenter, b.C6 as latency " +  
-       "FROM requests a JOIN metrics b ON a.C0  = b.C3) joinedtable " +   
-    "GROUP BY datacenter");
-joinDataFrame.ShowSchema();
-joinDataFrame.Show();
-```
-A simple DataFrame application using DataFrame DSL may look like the following
-```  c#
-// C0 - guid, C1 - datacenter
-var reqDataFrame = sqlContext.TextFile(@"hdfs://path/to/requests.csv")  
-                             .Select("C0", "C1");    
-// C3 - guid, C6 - latency   
-var metricDataFrame = sqlContext.TextFile(@"hdfs://path/to/metrics.csv", ",", false, true)
-                                .Select("C3", "C6"); //override delimiter, hasHeader & inferSchema
-var joinDataFrame = reqDataFrame.Join(metricDataFrame, reqDataFrame["C0"] == metricDataFrame["C3"])
-                                .GroupBy("C1");
-var maxLatencyByDcDataFrame = joinDataFrame.Agg(new Dictionary<string, string> { { "C6", "max" } });
-maxLatencyByDcDataFrame.ShowSchema();
-maxLatencyByDcDataFrame.Show();
-```
-Refer to `SparkCLR\csharp\Samples` directory for complete samples
-
-## Documents
-Refer to the docs @ https://github.com/Microsoft/SparkCLR/tree/master/docs
-
-## Building SparkCLR
-### Prerequisites
-* [Apache Maven](http://maven.apache.org) for spark-clr project implemented in scala
-* MSBuild in Visual Studio 2013 and above
-* .NET Framework 4.5 and above
-* [Nuget command-line utility](https://docs.nuget.org/release-notes) 3.2 and above
-
-### Instructions
-* Navigate to `SparkCLR\scala` directory and run the following command to build spark-clr*.jar   
-        ```
-        mvn package
-        ```
-* Start Developer Command Prompt for Visual Studio, navigate to `SparkCLR\csharp` directory, run the following commands to add `nuget.exe` to the path  
-        ```  
-        set PATH=<fullpath to nuget.exe>;%PATH%  
-        ```  
-        And build the rest of .Net binaries  
-        ```  
-        build.cmd  
-        ```  
-* Optional. Under `SparkCLR\csharp` directory, run the following command to clean the .NET binaries built above  
-        ```
-        clean.cmd
-        ```   
-
-## Running Samples
-### Prerequisites
-DataFrame TextFile API uses `spark-csv` package to load data from CSV file. Latest [commons-csv-*.jar](http://commons.apache.org/proper/commons-csv/download_csv.cgi) and [spark-csv*.jar (Scala version:2.10)](http://spark-packages.org/package/databricks/spark-csv) should be downloaded manually.
-
-The following environment variables should be set properly:
-* ```JAVA_HOME```  
-* ```SCALA_HOME```  

```

Comment:
SCALA_HOME disappeared when rendered by github - https://github.com/jthelin/SparkCLR/blob/README/README.md

It seems replace {```} with {```} fixed - e.g. - https://github.com/danielli90/SparkCLR/blob/mydev/README.md